app/src/main/java/ru/practicum/android/diploma/data/network/impl/NetworkClientImpl.kt

```diff
@@ -83,4 +91,16 @@ class NetworkClientImpl(
             resultCode = errorCode
         }
     }
+
+    private inline fun <T, reified R : Response> retrofit2.Response<T>.castToCustomResponse(
```

Comment:
Лучше импортнуть и написать так:
```suggestion
    private inline fun <T, reified R : Response> RetrofitResponse<T>.castToCustomResponse(
```

Сам импорт такой:
`import retrofit2.Response as RetrofitResponse`